### PR TITLE
new composer json for our gravity satis repo..

### PR DESCRIPTION
**Pull request by narcoticfresh**
[see title]

as discussed; this goes into develop now..

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "libgraviton/swagger-ui",
+  "license": "GPL",
```
Comment (hairmare) [inline composer.json]: Should be Apache-2 as it seems...

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "type": "project",
+  "description": "Graviton specific fork of swagger-ui",
+  "authors": [
+    {
+      "name": "List of contributors",
+      "homepage": "https://github.com/libgraviton/graviton/graphs/contributors"
```
Comment (hairmare) [inline composer.json]: Should be https://github.com/libgraviton/swagger-ui/graphs/contributors

**Comment by narcoticfresh** [comment]
OK; added..

**Comment by narcoticfresh** [comment]
@hairmare I will poor-man-merge this into `master` first and then open a PR for satis repo.. the upstream merge PR is separate; i'll open a PR again then we'll do a 'release'..

I'm just doing this so i have my feature branch available via satis repo so I can check it with my local graviton wrapper..

**Comment by narcoticfresh** [comment]
@hairmare travis is failing - should the build work? did it ever? it's new to me that we're having travis here too ;-)

**Comment by hairmare** [comment]
@narcoticfresh travis was prepared to push this to the cloud.... It can probably be ignored since we are switching that to jenkins anyhow... I'll try looking into why this fails so early nevertheless....

**Comment by narcoticfresh** [comment]
@hairmare ok.. can u check into this later - it doesn't affect the composer.json, but this PR would be nice so i can check my feature branch locally ;-)

**Pull request merged**